docs/modules/EMS_SolarLog.md

```diff
@@ -0,0 +1,28 @@
+# SolarLog EMS Module
+
+## Introduction
+
+The SolarLog EMS Module allows energy generation and production to be fetched from the Solar-Log Base API (```https://www.solar-log.com/de/produkte-komponenten/solar-logTM-hardware/solar-log-base/```) that generates a webservice directly on the device
+
+## Configuration
+
+The following table shows the available configuration parameters for the SolarLog EMS Module:
+
+| **Parameter** | **Value** |
+| ------------- | --------- |
+| enabled       | *required* Boolean value, ```true``` or ```false```. Determines whether we will poll the SolarLog API |
+| serverIP      | *required* The IP Address of the Solar-Log Base device. |
+| excludeConsumptionInverters        | *optional* Index of a reading device - to exclude consumption from. Needed if e.g. a boiler only heats with solar overhead - and you want to override this one. |
+
+Please note, if any of the required parameters for the SolarLog EMS module are not specified in the module configuration, the module will unload at start time.
+
+## JSON Configuration Example
+
+```
+"SolarLog": {
+  "enabled": false,
+  "serverIP": "192.168.1.2",
+  "excludeConsumptionInverters": "2"
```

Comment:
From your code, it looks like you expect this to be a comma-separated list.  An array might be clearer, and if they're integers you can skip the quotes.

Either way, I'd have multiple values in the example if you support it; it's easy to simplify a multi-value example to a single value, but generalizing a single example to multiple values requires reading code.

Reply:
Completely agree. Just updated the branch.